### PR TITLE
78 create connected server

### PR DESCRIPTION
**Pull request by jepoorbaugh**
Adds the flask server which will be used to create the API for the database which will power student analytics

### CODE REVIEW

File: app/db/recreate_database.sql
Comment (jncraton) [inline app/db/recreate_database.sql:1]: This file doesn't appear to be related to this PR, and it should not be included. There may be significant discussion about the DB schema, and we don't want to combine that with this other work.

File: www/student.html
Comment (jncraton) [inline www/student.html:1]: I don't believe the changes to this file are meaningful, and they should not need to be included.

File: .github/workflows/test_flask.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Automated testing
```
Comment (jncraton) [inline .github/workflows/test_flask.yml:1]: If we give this a different name (e.g. Flask Tests) it will be clearer which test is which when we look at the checks on Github.

**Review by owenmock22 — APPROVED** [review]

**Review by jncraton — APPROVED** [review]
